test/k8sT/Services.go

```diff
@@ -238,7 +241,23 @@ var _ = Describe("K8sServicesTest", func() {
 				data, err := res.Filter(`{.status.status.policy.realized.cidr-policy.egress}`)
 				ExpectWithOffset(1, err).To(BeNil(), "unable to get endpoint %s metadata", podName)
 				return data.String()
-			}, 2*time.Minute, 5*time.Second).Should(ContainSubstring(expectedCIDR))
+			}, 2*time.Minute, 2*time.Second).Should(ContainSubstring(expectedCIDR))
+		}
+
+		validateEgressAfterDeletion := func() {
+			By("Checking that toServices CIDR is no longer plumbed into CEP")
+			ExpectWithOffset(1, kubectl.WaitCEPReady()).To(BeNil(), "Cep is not ready after timeout")
+			Eventually(func() string {
+				res := kubectl.Exec(fmt.Sprintf(
+					"%s -n %s get cep %s -o json",
+					helpers.KubectlCmd,
+					helpers.DefaultNamespace,
+					podName))
+				res.ExpectSuccess("cannot get Cilium Endpoint")
```

Comment:
ExpectWithOffset(1, res).Should(helpers.CMDSuccess(), "cannot get Cilium endpoint")